tubesize/src/styles/toast.css

```
@@ -0,0 +1,86 @@
+    font-weight: 500;
+    text-align: center;
+    display: inline-flex;
+}
+.actions {
+    display: flex;
+    align-items: center;
+    gap: 4px;
+    flex-wrap: wrap;
+}
+button {
+    background: #ff416c;
```

Stylelint (17.7.0)

[error] Duplicate property "background" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)